CMakeLists.txt

```diff
@@ -8,11 +8,17 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 find_package(Catch2 REQUIRED)
 
-add_executable(test_something test_something.cpp)
-add_test(test_something test_something)
-target_link_libraries(test_something PRIVATE Catch2::Catch2)
+add_executable(main main.cpp)
 
-add_custom_target(  
+file(GLOB tests "${PROJECT_SOURCE_DIR}/test_*.cpp")
+foreach(test ${tests})
```

Comment:
I don't remember if this follows the naming we used to use: variable for test name and test file...